### PR TITLE
Feature/feature notifications

### PR DESCRIPTION
**Pull request by enrichp**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/notification/Notification.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5329711&bid=17444148)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java [hunk 1]
```diff
@@ -756,6 +770,50 @@ private String getImagePath(@Nullable Intent data) {
...
     }
 
+    /**
+     * Schedule a notification.
+     * @param context The context
+     * @param text The notification's text
+     * @param title The notification's title
+     * @param time The alarm time of the notification
+     */
+    @RequiresApi(api = Build.VERSION_CODES.M)
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java:780]: Which method needs to be in Android version 6.0 (Marshmallow)?

**Review by AlbertPG — CHANGES_REQUESTED** [review]
Take a look at the comment, it's important

**Review by AlbertPG — APPROVED** [review]
Take note of the issue

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java [hunk 2]
```diff
@@ -126,7 +134,10 @@
     private TrNewPersonalEvent trNewPersonalEvent;
     private TrDeletePersonalEvent trDeletePersonalEvent;
     private FloatingActionButton flAddCalendarEvent;
+    private static int NOTIFICATIONID;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/MainActivity.java]: Should be in camel case

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/notification/Notification.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5329711&bid=17444148)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/notification/Notification.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5329711&bid=17444148)

**Review by santidrj — APPROVED** [review]
👍

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/features/notification/Notification.java  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5329711&bid=17444148)

**Pull request merged**